### DISCUSSION TITLE
Viewing the time-averaged heat flux in SMV

### DISCUSSION THREAD

**Discussion by Tbuttts21** [firemodels/fds#12175] [General]
When I want to get pictures of the heat flux to targets, I typically just open the SMV file and choose a screenshot at some point throughout the simulation. If I wanted to get a picture of the time-averaged heat flux on all surfaces, is there a way to do that in SMV? 

I assume that this is not possible, so I was going to try and alter the files containing the heat flux data manually so the time-averaged heat flux shows up in SMV. However, I can't seem to figure out which files contain the heat flux data and how/if I can change the data within it.

**Comment by gforney** [comment]
you can time average slice files.

1. select Dialogs>Files/Data/Coloring dialog box
2. select Slice tab
3.  select Average data tab under slice
4. select the time interval you wish to average data over the load your
slice file
note.  the average does not take effect until after you load the data

On Fri, Oct 6, 2023 at 7:55 AM Tbuttts21 ***@***.***> wrote:

> When I want to get pictures of the heat flux to targets, I typically just
> open the SMV file and choose a screenshot at some point throughout the
> simulation. If I wanted to get a picture of the time-averaged heat flux on
> all surfaces, is there a way to do that in SMV?
>
> I assume that this is not possible, so I was going to try and alter the
> files containing the heat flux data manually so the time-averaged heat flux
> shows up in SMV. However, I can't seem to figure out which files contain
> the heat flux data and how/if I can change the data within it.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firemodels/fds/discussions/12175>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC6UCRT37CVXX2RZLTEU2UTX57WSZAVCNFSM6AAAAAA5VWQ3DSVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVG4YDOOBQGY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Glenn Forney

**Reply by Tbuttts21** [reply]
I know all of the average heat flux values I have for my targets, but I don't think SMV has the option to display the average heat flux values on each target being tracked. 

For example, I have a wall that is exposed to 30 kW/m^2 on average, but the only way I can grab a picture is by finding an instantaneous heat flux on the wall.
